Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANSILLAutoProcess.py

```diff
@@ -441,6 +447,12 @@ def PyExec(self):
 
         self.setProperty('OutputWorkspace', mtd[self.output])
         if self.output_sens:
+            GroupWorkspaces(InputWorkspaces=sensitivity_outputs, OutputWorkspace=self.output_sens)
+            if self.getProperty('SensitivityWithOffsets').value:
+                tmp_group_name = self.output_sens + '_group'
+                RenameWorkspace(InputWorkspace=self.output_sens, OutputWorkspace=tmp_group_name)
+                CalculateEfficiency(InputWorkspace=tmp_group_name, MergeGroup=True, OutputWorkspace=self.output_sens)
+                DeleteWorkspace(Workspace=tmp_group_name)
```

Comment:
There, I rename this group because `CalculateEfficiency` will create a workspace with the same name as the group, which does not override the group with all its entries, but only assigns a new workspace to the same name, while the group entries persist. With the rename and later delete, the group entries are removed and there is no unnecessary clutter in the ADS. If the efficiency takes priority, I can remove the `DeleteWorkspace` call.

Reply:
That's fine, I see now.